requirements.txt

```diff
@@ -6,4 +6,9 @@ svglib~=1.4.1
 svgwrite~=1.4.3
 entmax==1.0
 numba==0.56.4
-wmi~=1.5.1
\ No newline at end of file
+wmi~=1.5.1
+sparsemax==0.1.19
+entmax==1.1.0
+svglib==1.5.1
+svgwrite==1.4.3
+pyyaml==6.0.0
```

Comment:
The libraries listed in the pull request are already included in the `requirements.txt` except for `pyyaml`. Please, remove all duplicates